app/main.py

```diff
@@ -1,3 +1,52 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> repr:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float | Distance) -> float | Distance:
+        if isinstance(other, (int, float)):
+            km = self.km + other
+        else:
+            km = self.km + other.km
+        return Distance(km=km)
+
+    def __iadd__(self, other: int | float | Distance) -> float | Distance:
+        if isinstance(other, (int, float)):
+            self.km += other
+        else:
+            self.km += other.km
+        return self
+
+    def __mul__(self, other: int) -> Distance:
+        return Distance(
+            km=self.km * other
+        )
+
+    def __truediv__(self, other: float) -> Distance:
+        km = round(self.km / other, 2)
+        return Distance(
+            km=km
+        )
+
+    def __lt__(self, other: int) -> bool:
```

Comment:
other can be float or an instance of the Distance class.

Add logic to handle cases when other is an instance of the Distance class

fix it in all the methods below as well